modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java

```diff
@@ -126,23 +121,10 @@ protected void waitUntilContainerStarted() {
         // Repeatedly try and open a connection to the DB and execute a test query
 
         logger().info("Waiting for database connection to become available at {} using query '{}'", getJdbcUrl(), getTestQueryString());
-        Unreliables.retryUntilSuccess(getStartupTimeoutSeconds(), TimeUnit.SECONDS, () -> {
-
-            if (!isRunning()) {
-                throw new ContainerLaunchException("Container failed to start");
-            }
-
-            try (Connection connection = createConnection("")) {
-                boolean success = connection.createStatement().execute(JdbcDatabaseContainer.this.getTestQueryString());
 
-                if (success) {
-                    logger().info("Obtained a connection to container ({})", JdbcDatabaseContainer.this.getJdbcUrl());
-                    return null;
-                } else {
-                    throw new SQLException("Failed to execute test query");
-                }
-            }
-        });
+        await().ignoreExceptionsMatching(e -> ! (e instanceof NoDriverFoundException))
```

Comment:
```suggestion
        await().ignoreExceptionsMatching(e -> !(e instanceof NoDriverFoundException))
```

Reply:
I think I'd prefer to keep the space so that the `!` is more obvious, TBH